LML.NPOManagement.Bll/Services/NotificationService.cs

```diff
@@ -246,38 +246,41 @@ public async void EmailVerificationRequest(UserModel user)
 
         }
 
-        public async void EmailVerificationConfirmation(UserModel user)
+        public async void EmailVerificationConfirmation(UserModel user, string lang)
         {
             if (user == null)
             {
                 return;
             }
             var userInfo = user.UserInformations.FirstOrDefault();
 
-            var template = await GetTemplateByFileName("RegistracionNotification.html");
+            var template = await GetTemplateByFileName("RegistracionNotification.html", lang);
 
             template = template.Replace("@firstName", userInfo.FirstName);
             template = template.Replace("@lastName", userInfo.LastName);
             SendNotification(template, "Email Verification", user.Email);
 
         }
 
-        private async Task<string> GetTemplateByFileName(string templateName)
+        private async Task<string> GetTemplateByFileName(string templateName, string lang)
         {
+            var validLangs = new HashSet<string> { "am", "en" };
```

Comment:
Every time we add new lang we have to modify the codebase, not good. Here the ideas:
1. Supported languages are configured and you initialize your logic based on the configuration.
2. Supported languages are read dynamically from the folder structure (this is more complex and probably unnecessary, but also an option)

Reply:
this too can be moved to configuration file